server/memphis_handlers_schemas.go

```diff
@@ -17,6 +17,7 @@ import (
 	"time"
 
 	"github.com/gin-gonic/gin"
+	"github.com/graphql-go/graphql/language/parser"
```

Comment:
Since the name of this package is parser and we are using this name in other places (line 42) in this file I suggest to change the name of one of them

Reply:
Added alias name graphqlParser in import statement